src/main.js

```diff
@@ -48,15 +48,11 @@ if (window.location.search.indexOf("testEnvironment") > -1) {
     });
 } else {
     /**
-     * hack for r.js optimization, move locale to another config call
-     *
-     * Use custom brackets property until CEF sets the correct navigator.language
-     * NOTE: When we change to navigator.language here, we also should change to
-     * navigator.language in ExtensionLoader (when making require contexts for each
-     * extension).
+     * Check to see if a language is set through search string, and if not default to 'en'
      */
+    var locale = window.location.search ? /locale=([^&]+)&?/.exec(window.location.search)[1] : "en";
```

Comment:
A couple of things:

1. `"en"` will only be used if `window.location.search` doesn't exist, not if the search fails. A failing search could break this code.
2.  Why are we defaulting to `"en"` instead of sticking with what brackets was doing originally? It's probably better to use this part of their logic instead: `(typeof (brackets) !== "undefined" ? brackets.app.language : navigator.language)`